### DISCUSSION TITLE
Is there a way to change the virtual device of Macbook Silicon M2 to network bridging mode? I can't find an option for this setting.

### DISCUSSION THREAD

**Discussion by yuebinzhan** [Genymobile/feedback#88] [Genymotion Desktop] [answered]
Is there a way to change the virtual device of Macbook Silicon M2 to network bridging mode? I can't find an option for this setting.

**Comment by jmaneyrol69** [comment, marked as answer]
@yuebinzhan Genymotion Desktop uses QEMU hypervisor on Apple Mac M series, and this feature has not been ported yet. Also, please note that you cannot use VirtualBox on Apple Mac M series.
